modules/rbac-creating-cluster-role.adoc

```diff
@@ -34,4 +34,3 @@ following command:
 ----
 $ oc create clusterrole podviewonly --verb=get --resource=pod
 ----
-endif::[]
```

Comment:
Is this an accidental removal? It looks to be required due to the ifdef on line 7

Reply:
Yes it was - good catch! I replaced the missing `endif`.